examples/bri-3/front-end/src/login/Login.js

```diff
@@ -1,38 +1,69 @@
 import axios from 'axios';
+import { React, useState } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Card, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-const handleLogin = async () => {
-  // TODO: handle situations like account switch in metamask or when wallet is initially locked
-  if (window.ethereum) {
-    const accounts = await window.ethereum.request({ method: 'eth_requestAccounts' });
-    const nonceDto = {
-      publicKey: accounts[0]
-    };
-    const nonce = await axios.post(`${process.env.REACT_APP_SRI_BACKEND}/auth/nonce`, nonceDto);
-    const signature = await window.ethereum.request({
-      method: 'personal_sign',
-      params: [nonce.data, accounts[0]],
-    });
 
-    const loginDto = {
-      message: nonce.data,
-      signature,
-      publicKey: accounts[0],
-    };
+export default function Login() {
+  const [data, setData] = useState({
+    publicKey: '',
+    token: ''
+  });
+
+  const handleLogin = async () => {
+    try {
+    // TODO: handle situations like account switch in metamask or when wallet is initially locked
+    if (window.ethereum) {
+      const accounts = await window.ethereum.request({ method: 'eth_requestAccounts' });
+      const nonceDto = {
+        publicKey: accounts[0]
+      };
+      const nonce = await axios.post(`${process.env.REACT_APP_SRI_BACKEND}/auth/nonce`, nonceDto);
+      const signature = await window.ethereum.request({
+        method: 'personal_sign',
+        params: [nonce.data, accounts[0]],
+      });
+
+      const loginDto = {
+        message: nonce.data,
+        signature,
+        publicKey: accounts[0],
+      };
 
-    const tokenRequest = await axios.post(`${process.env.REACT_APP_SRI_BACKEND}/auth/login`, loginDto);
+      const tokenRequest = await axios.post(`${process.env.REACT_APP_SRI_BACKEND}/auth/login`, loginDto);
 
-    // TODO: add toast library and show success/failure login with error messages
-    // TODO: add logged in user details on UI
-    if (tokenRequest?.data?.access_token) {
-      console.log(tokenRequest.data.access_token);
+      // [Done] TODO: add toast library and show success/failure login with error messages
```

Comment:
Please remove all TODO comments that have been completed in this PR

Reply:
👌